src/bin/wasm-tools/objdump.rs

```diff
@@ -65,40 +76,75 @@ impl Opts {
     }
 }
 
+#[derive(Debug, Default)]
+struct IndexSpace {
+    modules: u32,
+    components: u32,
+    processing: Vec<Encoding>,
+}
+
+impl IndexSpace {
+    pub fn inc_module(&mut self) {
```

Comment:
Nit: could probably just delete these methods since they're no longer complex enough to justify them?